docs/DafnyRef/Attributes.md

```diff
@@ -300,38 +300,32 @@ values that satisfy the constraint.
 
 ### 22.1.13. opaque {#sec-opaque}
 Ordinarily the body of a function is transparent to its users but
-sometimes it is useful to hide it. If a function `f` is given the
-`{:opaque}` attribute then Dafny hides the body of the function,
+sometimes it is useful to hide it. If a function `foo` or `bar` is given the
+`{:opaque}` attribute, then Dafny hides the body of the function,
 so that it can only be seen within its recursive clique (if any),
-or if the programmer specifically asks to see it via the `reveal_f()` lemma.
+or if the programmer specifically asks to see it via the statement `reveal foo(), bar();` or the expression `reveal foo(), bar(); ...`.
```

Comment:
Thanks for "un-documenting" the old `reveal_f()` lemmas. The official support for this feature is with the `reveal` statement, as you have described.

I think it's okay to say just "statement ..." (and omit "or the expression ..."). Technically, there is no special `reveal` _expression_. Instead, the `reveal` _statement_ is one of about 5 statements that are allowed to be part of a `StmtExpr`.